src/libraries/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj

```diff
@@ -1,10 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent)-Unix;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  <ItemGroup>
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <TargetPlatformIdentifier>$([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)'))</TargetPlatformIdentifier>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetPlatformIdentifier)' == ''">SR.PlatformNotSupported_FileSystemDriveInfo</GeneratePlatformNotSupportedAssemblyMessage>
```

Comment:
I wonder if we could treat the browser impl as the default here instead of PNSE?  It seems to be rather platform agnostic.  Where possible it will be nice to "save" additional configurations.  It helps with all-configurations build and helps with platform bring-up.  I didn't look through all libraries, but if you can find any other cases where an existing configuration makes sense to promote to RID-less, that might be a better option.

Reply:
Thanks, I agree that DriveInfo's Browser implementation is a good platform agnostic candidate. I renamed the files to indicate that they comprise the default implementation.